pkgs/applications/misc/subsurface/default.nix

```diff
@@ -23,7 +23,7 @@
   qtlocation,
   qtsvg,
   qttools,
-  qtwebengine,
```

Comment:
:+1: from my side

The impact should be rather small. According to subsurface's [CMakeLists.txt](https://github.com/subsurface/subsurface/blob/master/CMakeLists.txt#L283), this should only disable the internal manual and printing support.

We can re-enable `qtwebengine` once subsurface builds with [recent Qt 6 versions.](https://github.com/subsurface/subsurface/issues/4576)

Reply:
on the version we have, qtwebengine is not used *at all*

Reply:
Indeed, seems like the cmake cannot find it. This should be the [code](https://github.com/subsurface/subsurface/blob/master/CMakeLists.txt#L290).

Reply:
Careful: we are on commit 38a0050ac33566dfd34bf94cf1d7ac66034e4118, which only supports qtwebkit: https://github.com/subsurface/subsurface/blob/38a0050ac33566dfd34bf94cf1d7ac66034e4118/CMakeLists.txt#L249-L275

qtwebkit is the predecessor to qtwebengine, and *even worse* in terms of security, see e.g. https://blogs.gnome.org/mcatanzaro/2022/11/04/stop-using-qtwebkit/

qtwebengine is not supported at all for the version we have. Not even for qt6, where it is just blanket-disabled. We'd need to update the package to use a modern version *and* qt6 to make use of qtwebengine.

Reply:
This commit is interesting :)

https://github.com/subsurface/subsurface/commit/643f4a5726926b86693b442e318f40cf28fee04a